pom.xml

```diff
@@ -148,72 +156,71 @@
     <plugins>
       <!-- Ensures checksums are added to published jars -->
       <plugin>
-	<artifactId>maven-install-plugin</artifactId>
-	<version>${maven-install-plugin.version}</version>
-	<configuration>
-	  <createChecksum>true</createChecksum>
-	</configuration>
+      	<artifactId>maven-install-plugin</artifactId>
```

Comment:
Lets avoid these changes the next time - usually style changes add noise only (they can happen independently if/as needed, though ;) )

Reply:
Sorry about that.